bbc/simorgh
# Fix UGC test type errors

Overall changes
======
- Fixes UGC component test type errors as they were creating noise when running `tsc` in the `ws-nextjs-app` directory

Testing
======
1. Navigate to the `ws-nextjs-app` directory
2. Run `tsc` in Terminal
3. Confirm there are no TS errors returned

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
